porcupine/plugins/directory_tree.py

```diff
@@ -126,27 +137,15 @@ def _on_click(self, event: tkinter.Event[DirectoryTree]) -> str | None:
         if not item:
             return None
 
-        # Couldn't get <Double-Button-1> to work, so I wrote a program to
-        # measure max time between double clicks. It's 500ms on my system.
-        double_click = event.time - self._last_click_time < 500 and self._last_click_item == item
-
         self.set_the_selection_correctly(item)
-        if item.startswith("file:"):
-            if double_click:
+
+        little_arrow_clicked = self.identify_element(event.x, event.y) == "Treeitem.indicator"
+
+        if little_arrow_clicked or not item.startswith("file"):
```

Comment:
I don't think we need `little_arrow_clicked` anymore, because directories open regardless of where you click, and files don't have the arrow.